src/components/CommentsOnArticle.jsx

```diff
@@ -0,0 +1,42 @@
+import { useParams} from 'react-router-dom'
+import { useEffect, useState } from "react"
+import { getCommentsOnArticle, getArticleById } from './utils/api'
+import CommentCard from './CommentCard'
+
+const CommentsPage = () => {
+    const {article_id} = useParams()
+    const [commentsOnArticle, setCommentsOnArticle] = useState([])
+    const [specificArticle, setSpecificArticle] = useState([])
+            useEffect(() => {
+            getArticleById(article_id).then((article) => {
+                setSpecificArticle(article)
+            })
+        }, [])
```

Comment:
The comments being on a different page/route mean that you've had to make this database request again which has already been done on the previous page to this.